packages/babel-plugin/src/__perf__/module-traversal-caching.tsx

```diff
@@ -0,0 +1,66 @@
+import Benchmark, { Event as BenchmarkEvent } from 'benchmark';
+import { transformSync } from '@babel/core';
+
+import babelPlugin from '../index';
+
+const suite = new Benchmark.Suite('module-traversal-cache');
+
+const transform = (code: string, options = { cache: false }) => {
+  return transformSync(code, {
+    configFile: false,
+    babelrc: false,
+    compact: true,
+    highlightCode: false,
+    filename: process.cwd() + '/src/__perf__/module-traversal-caching.js',
+    plugins: [[babelPlugin, options]],
+  })?.code;
+};
+
+suite
+  .add('#initial-run', () => {
+    transform(
+      `
+      import '@compiled/core';
+      import React from 'react';
+
+      import { colorMixin } from '../__tests__/stubs/objects';
```

Comment:
Can we move these stubs outside of `__tests__` folder? Probably in `__fixtures__` folder under `src` to make them more reusable.

Reply:
Sounds good